### PR TITLE
Updates the version of the translator and corrects XML vs JSON handling

### PR DESCRIPTION
**Pull request by brynrhodes**

### CODE REVIEW

File: src/main/java/org/mitre/bonnie/cqlTranslationServer/TranslationResource.java [hunk 1]
```diff
@@ -70,7 +70,11 @@ public Response cqlPackageToElmPackage(FormDataMultiPart pkg,
       for (String fieldId: pkg.getFields().keySet()) {
         for (FormDataBodyPart part: pkg.getFields(fieldId)) {
           CqlTranslator translator = getTranslator(part.getEntityAs(File.class));
-          translatedPkg.field(fieldId, translator.toJson(), targetFormat);
+          if (targetFormat.equals(new MediaType("application", "elm+xml"))) {
```
Comment (hadleynet) [inline src/main/java/org/mitre/bonnie/cqlTranslationServer/TranslationResource.java]: Suggest using the ELM_XML_TYPE constant instead

**Review by hadleynet — CHANGES_REQUESTED** [review]

File: src/main/java/org/mitre/bonnie/cqlTranslationServer/TranslationResource.java (hunk 1 above)
Comment (hadleynet) [reply src/main/java/org/mitre/bonnie/cqlTranslationServer/TranslationResource.java]: MediaType.valueOf(ELM_XML_TYPE)

Comment (brynrhodes) [reply src/main/java/org/mitre/bonnie/cqlTranslationServer/TranslationResource.java]: Agreed and applied.

File: Dockerfile [hunk 2]
```diff
@@ -19,4 +19,4 @@ EXPOSE 8080
 
 # execute it
 # CMD ["mvn", "exec:java"]
-CMD ["java", "-jar", "target/cqlTranslationServer-1.2.0-SNAPSHOT-jar-with-dependencies.jar", "-d"]
+CMD ["java", "-jar", "target/cqlTranslationServer-1.2.1-SNAPSHOT-jar-with-dependencies.jar", "-d"]
```
Comment (hadleynet) [inline Dockerfile]: Shouldn't this be 1.2.8 to match the pom update?

Comment (brynrhodes) [reply Dockerfile]: Yes, I missed this, thanks! Just committed the fix.

**Pull request merged**